corehq/apps/export/views.py

```diff
@@ -973,7 +975,20 @@ def dispatch(self, request, *args, **kwargs):
         if not (self.has_edit_permissions or self.has_view_permissions
                 or (self.is_deid and self.has_deid_view_permissions)):
             raise Http404()
-        return super(BaseExportListView, self).dispatch(request, *args, **kwargs)
+
+        self.request = request
+
+        if domain_has_daily_saved_export_access(self.domain) or domain_has_excel_dashboard_access(self.domain):
```

Comment:
I think you also want to do this check:
https://github.com/dimagi/commcare-hq/blob/2d42bbf3088f7ac1ec7a881df1f7f4d27ca5909c/corehq/apps/export/views.py#L1511